bitcoin/src/taproot/mod.rs

```diff
@@ -799,6 +799,30 @@ pub struct NodeInfo {
     pub(crate) has_hidden_nodes: bool,
 }
 
+/// Explicitly implement Ord so future changes to NodeInfo (e.g. adding a new field) won't result in
+/// potentially changing addresses out from under users
+impl Ord for NodeInfo {
+    fn cmp(&self, other: &Self) -> core::cmp::Ordering {
```

Comment:
nit: We can reduce the line count by 5 lines by importing `core::cmp::Ordering` (and running the formatter).

Reply:
done 👍🏼